yargs.js

```diff
@@ -204,7 +204,7 @@ function Yargs (processArgs, cwd, parentRequire) {
 
   self.requiresArg = function (keys) {
     argsert('<array|string>', [keys], arguments.length)
-    populateParserHintArray('requiresArg', keys)
+    populateParserHintObject(self.nargs, false, 'narg', keys, 1)
```

Comment:
Is calling `self.nargs(keys, 1)` too cute, here?

Both ways seem fine, the tradeoff being a duplicated call to `populateParserHintObject` vs a redundant `argsert` in the nested call...

Reply:
@evocateur it doesn't seem too cute; I like your idea of calling `self.nargs`.

Reply:
I was envisioning it being the entire contents of the method, not in addition to the duplicate argsert+populate call.